### ISSUE TITLE
Doesn't work for the Calendar integrated as a Sidepanel for example in Gmail 

### ISSUE THREAD

**Issue by TheMen4ce** [imightbeamy/gcal-multical-event-merge#79]
The feature works great on calendar.google.com but not when having the calendar opened in GMail as a sidepanel

![image](https://user-images.githubusercontent.com/9498649/70225469-61ddc300-174f-11ea-975f-0fb086ff2fee.png)

Chrome version 78.0.3904.108 (Official Build) (64-bit)
on Windows 10 1909

**Comment by torzsmokus** [comment]
duplicate of #75 ?

**Comment by TheMen4ce** [comment]
> duplicate of #75 ?

yes it is. but the other one is closed. And it still doesn't work ;)
